edb/server/compiler/compiler.py

```diff
@@ -1449,8 +1506,10 @@ def _compile_ql_administer(
             )
 
         return dbstate.MaintenanceQuery(sql=(b'ANALYZE',))
-    elif ql.expr.func == 'repair_schema':
+    elif ql.expr.func == 'schema_repair':
         return ddl.administer_repair_schema(ctx, ql)
+    elif ql.expr.func == 'object_reindex':
```

Comment:
Why `object_reindex` and not just `reindex`?  Is this somehow specific to object relations or should also apply to link relations (e.g. indexes on link properties).